app/javascript/hyacinth_v1/components/digital_objects/DigitalObjectInterface.js

```diff
@@ -1,22 +1,53 @@
 import React from 'react';
 import { capitalize } from 'lodash';
 import PropTypes from 'prop-types';
+import { useHistory } from 'react-router-dom';
 
 import Tab from '../shared/tabs/Tab';
 import Tabs from '../shared/tabs/Tabs';
 import TabBody from '../shared/tabs/TabBody';
 import ContextualNavbar from '../shared/ContextualNavbar';
+import { Button } from 'react-bootstrap';
 import DigitalObjectSummary from './DigitalObjectSummary';
+import { redirectToSearch } from '../../utils/redirectToSearch';
 
 function DigitalObjectInterface(props) {
   const { digitalObject, children } = props;
   const { id, title, digitalObjectType } = digitalObject;
+  const history = useHistory();
+  const latestSearchQueryString = sessionStorage.getItem('searchQueryParams');
+
+
+  const backToSearchHandler = () => {
+    const latestSearchQuery = JSON.parse(latestSearchQueryString);
+    redirectToSearch(history, latestSearchQuery);
+  };
+
+  const backToSearchButton = () => {
+    const button = latestSearchQueryString ?
+          <div>
+                <Button
+                  variant="Dark"
```

Comment:
```suggestion
                  variant="dark"
```

Reply:
Actually, for semantic html reasons (and for proper accessibility, and for consistent UI styling and hover state in the navbar) we should use an <a> tag here instead of a button because it leads to a different URL.  I was about to suggest just using an <a> tag here in the JSX, but just learned that the react bootstrap library automatically renders an <a> if you include an href attribute on the button (https://react-bootstrap.github.io/components/buttons/#button-tags).  This would mean an additional change to the `backToSearchHandler` situation: you'd want a method in the same module that actually returns the new url (e.g. `backToSearchPath()`) rather than performing the history update.

And then you would change the ContextualNavbar render to:
```
<ContextualNavbar
        title="Create Batch Import"
        rightHandLinks={[
          { link: backToSearchPath, label: 'Back to Search' },
        ]}
      />
```

And the `backToSearchButton` method can go away altogether.